be/src/vec/exec/volap_scanner.h

```diff
@@ -112,6 +112,7 @@ class VOlapScanner {
     std::vector<bool> _runtime_filter_marks;
 
     int _id;
```

Comment:
is this  field useful?

Reply:
I think it would be helpful when debugging, cause the id of volapscanner always be -1 now, maybe  it should be marked as compile in  debug only...

Reply:
deleted _id_gen